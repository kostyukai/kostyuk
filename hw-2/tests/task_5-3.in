2 21 30 4 10
1.0 2.0 3.5 4.0 5.1
