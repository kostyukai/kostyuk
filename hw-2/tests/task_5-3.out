67
15.60
